gapic/src/main/com/google/gapid/util/Colors.java

```diff
@@ -42,6 +44,18 @@ public static RGB rgb(double r, double g, double b) {
     return new RGB((int)(r * 255), (int)(g * 255), (int)(b * 255));
   }
 
+  public static RGB getRandomColor(int index) {
+    float hue = (float)Math.toDegrees(index * GOLDEN_ANGLE) % 360;
+    return new RGB(hue, 1f, 1f);
```

Comment:
I actually spent quite a lot of time trying to get colors that were non-offensive.
The code I wrote is not a simple HSV to RGB function. Unless there's a good justification, I'd prefer it if you kept the original code.